wp-content/themes/haruoshi_origin/src/scss/single-works.scss

```
@@ -0,0 +1,29 @@
+@import "variables";
+@import "mixin";
+
+.container {
+	inset: 0;
+	margin: $header-height auto 60px;
+	max-width: $contents-max-width;
+	width: calc(100% - 20px);
+	&__title {
+		font-size: 24px;
+		font-weight: bold;
+	}
+	&__date {
+		margin: 20px 0;
+	}
+	&__thumbnail {
+		max-width: $contents-max-width;
+		width: calc(100% - 20px);
```

一応補足情報として、IEだとautoでもいいのでheightを指定しないと画像の高さが0になってしまいます。今回はIEはサポート対象外という想定で実装しているので修正しなくても大丈夫です。
```suggestion
		width: calc(100% - 20px);
		height: auto; // IE対応のために指定
```

---

done https://github.com/oooshima/origin_wordpress/pull/28/commits/0f30841f9db13daeae962309b650d316ffaf75af